benchmarks/bench_evolve.py

```diff
@@ -34,8 +34,12 @@ def benchmark_write_operator_card_from_file(tmp_path, test_files, test_configs):
     assert np.allclose(myopcard["rotations"]["xgrid"], x_grid)
     assert np.allclose(myopcard["rotations"]["targetgrid"], x_grid)
     assert np.allclose(myopcard["rotations"]["inputgrid"], x_grid)
```

Comment:
Drop also `"targetgrid"` and `"inputgrid"`, they are as useless as the other two now :)

Reply:
I believe that @felixhekhorn thinks that they are not useless and that `eko` would crash without them (and even without `targetpids` and `inputpids`). I am not an expert of `eko` so I wait for you

Reply:
But here you are dropping `targetpids` and `inputpids`, so if you are dropping them, you can drop also the other two :)